### PR TITLE
Refactor cluster settings fe

### PR DESCRIPTION
**Pull request by fabriziosestito**
This PR removes consul completely from the web controllers by refactoring `/checks/:id/settings` leveraging the new methods on the cluster controller.
A follow-up cleanup is needed to nest the APIs under the clusters resource.

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -14,7 +14,6 @@ import (
 
 type TrentoApiService interface {
 	IsWebServerUp() bool
-	GetChecksSettingsById(id string) (*webApi.JSONChecksSettings, error)
```
Comment (nelsonkopliku) [inline api/api.go:17]: Ah great! Thanks. I should have cleaned up that.
Next to this we can also remove `api/checks.go` and `api/checks_test.go`, would you mind? :innocent:

File: web/checks_api.go [hunk 2]
```diff
@@ -1,24 +1,18 @@
...
-	"github.com/trento-project/trento/internal/hosts"
 	"github.com/trento-project/trento/web/models"
 	"github.com/trento-project/trento/web/services"
 )
 
 type JSONChecksSettings struct {
 	SelectedChecks     []string          `json:"selected_checks" binding:"required"`
 	ConnectionSettings map[string]string `json:"connection_settings" binding:"required"`
-	Hosts              Nodes             `json:"hosts"`
+	Hostnames          []string          `json:"hostnames"`
```
Comment (nelsonkopliku) [inline web/checks_api.go:15]: Do we need to also `make generate` to have swagger docs updated?

**Review by nelsonkopliku — COMMENTED** [review]
That's great! Just a couple of minor things.

File: web/checks_api.go (hunk 2 above)
Comment (fabriziosestito) [reply web/checks_api.go:15]: yes you right pushing it now

**Review by nelsonkopliku — APPROVED** [review]
Lovely

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**